### DISCUSSION TITLE
how nocodb docker set timezone

### DISCUSSION THREAD

**Discussion by chaochaokeji** [nocodb/nocodb#1443] [Q&A]
how nocodb docker set timezone?

**Comment by wingkwong** [comment]
It follows the timezone in a docker container. By default, it's UTC. If you want to change the timezone, you can specify it in ``TZ``.

Example:

```
docker run -d -e TZ=Asia/Hong_Kong --name nocodb -p 8080:8080 nocodb/nocodb:latest
```
